fastlane/Fastfile

```diff
@@ -110,10 +112,19 @@ platform :ios do
       configuration: "Release"
     )
 
+    app_store_connect_api_key(
+      key_id: ENV["FASTLANE_APPSTORE_CONNECT_KEY_ID"],
```

Comment:
Could you please provide the values of these env variables in a shared Bitwarden note? That would be great 🙏

Reply:
I wish I could @ecotopian . Actually tried it. Meant to reach out to you to provide you with the .p8 file